CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 ## CHANGE LOG
 
+### V0.8.0
+
+#### Updated
+
+- Updates React to version 17.
```

Comment:
wait really, you can do this on node 10?

Reply:
that's amazing

Reply:
I'm just as surprised. I'm thinking that it actually installs and uses React 16 but 🤷